src/Lightning.ts

```diff
@@ -603,7 +599,7 @@ export const LightningMixin = (superclass) =>
             }
 
             // success
-            await Transaction.updateMany({ hash: id }, { pending: false })
+            await ledger.settlePayment(id)
```

Comment:
I think we should either use pending or settle but not mix them. wdyt?

Reply:
what name do you prefer?  I couldn't use confirm because the same code is used for void

Reply:
I'm not sure what is better. I like settlePayment, but I think in that case we should have `settle: true` instead of `pending: false`. but this implied a db updates. maybe in another 
PR?

maybe in the meantime `setPending(false)`? not sure really